packages/pn-commons/src/types/ApiKeys.ts

```diff
@@ -0,0 +1,23 @@
+export interface ApiKey {
```

Comment:
@NicolaGiornetta rivediamo la modellazione di questa entity seguendo questo schema https://petstore.swagger.io/?url=https://raw.githubusercontent.com/pagopa/pn-delivery/develop/docs/openapi/api-external-b2b-pa-v1.yaml. Tipizzare status con ApiKeyStatus. Ricontrollare in generale i modelli